internal/jimm/relation.go

```diff
@@ -110,29 +110,60 @@ func (j *JIMM) ListObjectRelations(ctx context.Context, user *openfga.User, obje
 	if !user.JimmAdmin {
 		return nil, e, errors.E(op, errors.CodeUnauthorized, "unauthorized")
 	}
-	continuationToken, kind, err := pagination.DecodeEntitlementToken(entitlementToken)
+	responseTuples, nextToken, err := j.getObjectRelationsPage(ctx, object, pageSize, entitlementToken)
 	if err != nil {
-		return nil, e, err
+		return nil, e, errors.E(op, err)
 	}
+	// verify next page contains some entries. Otherwise return empty nextToken.
+	if len(responseTuples) == int(pageSize) && nextToken.String() != "" {
```

Comment:
should we care if the next page contains entries? or just keep returning if openfga returns a continuation token.. also i think the correct way would be to 
```
if len(responseTuples) > 0 && nextToken.String() != "" {}
```

Reply:
we care because otherwise frontend receives a meaningless token